cmd/cloudshell_open/cloudrun.go

```diff
@@ -32,10 +34,44 @@ const (
 	defaultRunMemory = "512Mi"
 )
 
+type service struct {
+	Spec serviceSpec `json:"spec"`
+	Status serviceStatus `json:"status"`
+}
+
+type serviceSpec struct {
+	Template serviceTemplate `json:"template"`
+}
+
+type serviceTemplate struct {
+	Spec templateSpec `json:"spec"`
+}
+
+type templateSpec struct {
+	Containers []container `json:"containers"`
+}
+
+type container struct {
+	EnvVars []envVar `json:"env"`
+}
+
+type envVar struct {
+	Name string `json:"name"`
+	Value string `json:"value"`
+}
+
+type serviceStatus struct {
+	Address address `json:"address"`
+}
+
+type address struct {
+	Url string `json:"url"`
+}
+
 func optionsToFlags(options options) []string {
-	authSetting := "--no-allow-unauthenticated"
-	if *options.AllowUnauthenticated {
-		authSetting = "--allow-unauthenticated"
+	authSetting := "--allow-unauthenticated"
+	if options.AllowUnauthenticated != nil && *options.AllowUnauthenticated == false {
```

Comment:
I think the PRs got mixed together. :) maybe a rebase will help.

Reply:
Moved out to another PR: https://github.com/GoogleCloudPlatform/cloud-run-button/pull/127